internal/db/schema/migrations/oss/postgres/52/01_worker_operational_state.up.sql

```diff
@@ -0,0 +1,76 @@
+begin;
```

Comment:
Might be good to get @mgaffney doing a quick check through this, he often finds subtle things that are easy to overlook.